src/proto/beats.proto

```diff
@@ -10,22 +11,75 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
-  optional int32 frame_position = 1;
-  optional bool enabled = 2 [ default = true ];
-  optional Source source = 3 [ default = ANALYZER ];
+enum SectionType {
+  INTRO = 0;
+  OUTRO = 1;
+  OTHER = 2;
+}
+
+// Setting grey as the default color
+message Color {
+  optional int32 red = 1 [ default = 100 ];
+  optional int32 green = 2 [ default = 100 ];
+  optional int32 blue = 3 [ default = 100 ];
 }
 
 message Bpm {
-  optional double bpm = 1;
-  optional Source source = 2 [ default = ANALYZER ];
+  optional double bpm = 1 [ default = 0 ];
```

Comment:
This default value is redundant: https://developers.google.com/protocol-buffers/docs/proto#optional